app/controllers/admin/paypal_preferences_controller.rb

```diff
@@ -58,6 +63,9 @@ def index
       admin_getting_started_guide_path,
       Admin::OnboardingWizard.new(@current_community.id).setup_status)
 
+    currencies = MarketplaceService::AvailableCurrencies::CURRENCIES
+    available_currencies = [currency].concat(currencies.select { |c| c != currency })
```

Comment:
That was originally there to put the current currency as the first value in the list, no matter what the sorting was. But I guess it doesn't make that much sense, especially since I now figured out how to define the selected value... :P